File: Learn C#/20230728/MyBankAppvb/Models/Movie.vb
Imports System.Data.Entity

Public Class Movie
    Public Property ID() As Integer
    Public Property Title() As String
    Public Property ReleaseDate() As Date
    Public Property Genre() As String
    Public Property Price() As Decimal
End Class

Public Class MovieDBContext
    Inherits DbContext
    Public Property Movies() As DbSet(Of Movie)
    Public Property BankAccounts As System.Data.Entity.DbSet(Of MyBankAppvb.Models.BankAccount)
    Public Property Transactions As System.Data.Entity.DbSet(Of MyBankAppvb.Models.Transaction)
End Class

File: Learn C#/20230725/MyBankApp/Models/Movie.cs
﻿using System;
using System.Collections.Generic;
using System.Data.Entity;

namespace MvcMovie.Models
{
    public class Movie
    {
        public int ID { get; set; }
        public string Title { get; set; }
        public DateTime ReleaseDate { get; set; }
        public string Genre { get; set; }
        public decimal Price { get; set; }
    }

    public class MovieDBContext : DbContext
    {
        public DbSet<Movie> Movies { get; set; }
    }
}